src/Console/stubs/custom-presenter.php.stub

```diff
@@ -11,8 +11,8 @@ class DummyClass extends Presenter
      *
      * @return string
      */
-    public function name()
+    public function name(): string
     {
-        return $this->model->name;
+        return ucwords($this->model->name);
     }
-}
\ No newline at end of file
+}
```

Comment:
@ErickZH 
Add the final line, please :)

Reply:
@zaratedev It is already included in this Pull Request

Reply:
Thanks :)